test/test_cluster_ray.py

```diff
@@ -0,0 +1,155 @@
+"""Ray-Horovod Job cluster tests.
+
+This is currently not run on the Ray CI and is expected
+to land in Horovod repo soon. If not, then these
+should be run after every commit for anything that touches
+the Horovod-Ray integration.
+"""
+
+import os
+
+import ray
```

Comment:
This file probably needs a little bit of clean-up; it's written in a way for multi-node setups but we can 1. get the pytest infrastructure up and 2. make it work on single-node setups first.